apps/src/Sounds.js

```diff
@@ -259,6 +259,16 @@ Sounds.prototype.play = function (soundId, options) {
   if (sound) {
     sound.play(options);
   }
+  this.audioStartedTime = this.audioContext.currentTime;
```

Comment:
I don't think this will work for HTML5 audio, where we don't have an `audioContext`, but we do have an `audioElement`.